src/nodedef.cpp

```diff
@@ -130,7 +115,7 @@ void NodeBox::serialize(std::ostream &os, u16 protocol_version) const
 void NodeBox::deSerialize(std::istream &is)
 {
 	int version = readU8(is);
-	if (version < 1 || version > 3)
+	if (version < 4 || version > 4)
```

Comment:
shouldn't this be `version < 4`?

Reply:
If we do that, we can't change the format easily in future versions. The server must send the data which the client (protocol version) asks for.